src/main/java/haste/ui/Ui.java

```diff
@@ -0,0 +1,60 @@
+package haste.ui;
+
+import haste.data.TaskList;
+import haste.tasks.Task;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+import static java.lang.Integer.parseInt;
+
+/**
+ * Represents a user interface that prints information ot user.
+ */
+public class Ui {
+    public static final String LINE = "＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿＿";
+    public static final String INDENT = "    ";
+
+    public boolean running = true;
+
+
+    public static void formatPrint(String input) {
+        System.out.println(INDENT + LINE);
+        String[] lines = input.split("\n");
+        for (String line : lines) {
+            System.out.println(INDENT + line);
+        }
+        System.out.println(INDENT + LINE);
+    }
+    public void greet() {
+        formatPrint("Hello! I'm HASTE\nWhat can I do for you?");
+    }
+    public void bye() {
+        formatPrint("Bye. Hope to see you again!");
+        this.running = false;
+    }
+
+    public void mark(String taskDesc, TaskList tasks) {
```

Comment:
I think you can consider using method names that are more specific to this class, eg. addMessage, markMessage etc. to avoid confusion with methods in other classes